### DISCUSSION TITLE
Weird stuff after installation

### DISCUSSION THREAD

**Discussion by GloryOfNight** [o3de/o3de#10842] [General]
After I installed O3DE from their Windows installer, my laptop temperatures constantly > 80*C and cooling would not shut up.
Firstly I was thinking that I'm paranoid and something else is going on, but, uninstalled o3de and everything was back to normal. 

A little later I decided to confirm the problem, I installed it again. After system reboot, the behavior has been replicated with same higher then normal temps and that's with 15+ mins of just sitting around doing supposedly nothing. 
I checked Download / Upload traffic from router and there is definitely something going on, behavior that not present with uninstalled o3de on my Windows machine.

Can someone explain?

**Comment by cpgames** [comment]
You most likely have asset processor running in background, check your taskbar icons on the bottom right (system tray).
